.gitignore

```diff
@@ -1,7 +1 @@
-*.swp
-*~
```

Comment:
Is there a particular reason these are deleted? They're useful for people using different editors.

Reply:
Sorry I just copied another .gitignore file. I'll add this back.